### ISSUE TITLE
[Request] ShowAllActivities

### ISSUE THREAD

**Issue by icrazykola** [Vencord/plugin-requests#258]
### Plugin Description

Shows all activities currently active on the profile popout, rather than only one.
Mockup: https://i.imgur.com/3UPrN8c.png

### Existing Plugin for other mod

BD has one: https://betterdiscord.app/plugin/ShowAllActivities but if anyone decides to work on this I would love to see a version like my mockup where all activities are shown at once rather than scrolling through all of them 🙏🙏

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by JustUsingGit** [comment]
@D3SOX is your plugin still active? I can't find it on plugins, and it would be very helpful having it, since I was searching for something like that.

**Comment by D3SOX** [comment]
@JustUsingGit It's currently only available via my userplugins, not inside Vencord, see https://github.com/Vendicated/Vencord/pull/2178 for the status

**Comment by JustUsingGit** [comment]
I see, so it hasn't been approved yet. Hope they will soon, then

**Comment by icrazykola** [comment]
@D3SOX Sorry I didn't see your comment at all hahaha, the reason I requested MLA was to be a kind of convenience tool to see the activities being played at a glance, so this was kind of a popout aesthetics/convenience of a convenience tool to see the details at a glance rather than popout > profile > activity all that 

If you do choose to work on it honestly half of it was the aesthetics, its aesthetics fits with the new "recent games" experiment so mostly it's just to look good :)

**Comment by waterbuffalowalking** [comment]
+1 for this from me
@D3SOX maybe you could add this as a setting to your plugin? MLA is only useable for people in member list, most people I click on are in chat

**Comment by D3SOX** [comment]
The thing is I already tried to find that component and mount it for the tooltips in MemberListActivities and failed. I don't really think it's worth the effort creating such a plugin when you can already see the same information in the member list, which is always there and doesn't need a click.

**Comment by waterbuffalowalking** [comment]
@D3SOX The issue is half the memberlist isn't even visible in big servers and scrolling would be a pain, this plugin would just be extremely Quality of Life because theres no reason at all to look in the Activity section of the Profile in the first place because half the time there's nothing else there anyways

**Comment by icrazykola** [comment]
@waterbuffalowalking That's honestly not a bad idea if Nico is up to it, could name the plugin something more broad like "ActivityUtils", but at the same time I think this would go better as a separate plugin since this and MLA aren't really that similar

**Comment by D3SOX** [comment]
I like the idea of having it one plugin. I made some progress mounting the activity component. Still need to fix the CSS tho

Tooltip:
![image](https://github.com/Vencord/plugin-requests/assets/24937357/0d002ae9-3b60-4b9a-a06d-8832bc4e2f2a)

DM Sidebar:
![image](https://github.com/Vencord/plugin-requests/assets/24937357/6e16eb10-25b2-4b35-b6fe-c4399293211e)

**Comment by icrazykola** [comment]
That already looks super good - if it's one plugin will they both be integrated or will they be toggleable? Either way I'd be using both so no difference to me hahaha

**Comment by D3SOX** [comment]
Yeah, I just renamed the plugin to BetterActivities and added toggles for the features. I'm going to work on the fix and popout implementation later.

**Comment by D3SOX** [comment]
I just saw that @AutumnVN already ported the ShowAllActivities plugin here:
https://github.com/AutumnVN/userplugins/tree/main/activities

@AutumnVN Can we maybe somehow combine our plugins because in my opinion yours does the showing all activities in the user popout better and mine the member list icon part.

**Comment by D3SOX** [comment]
Great news with the great help of @AutmnVN we integrated the ShowAllActivities functionality.
The plugin is now available in my userplugins as [BetterActivities](https://github.com/D3SOX/vencord-userplugins/tree/master/betterActivities)

**Comment by icrazykola** [comment]
That's so awesome!! Thanks for considering this :) In the end is there no way to mount both activities at the same time on the popout, ending with the carousel-type display?

**Comment by icrazykola** [comment]
Yeah the version on BD was a carousel as well, if it's possible at all to have all of them together like [this](https://i.imgur.com/3UPrN8c.png) that would seriously be amazing 😭♥

**Comment by D3SOX** [comment]
Just added an option for that :)

**Comment by icrazykola** [comment]
Thank you so much!! <3

**Comment by waterbuffalowalking** [comment]
this is amazing!
